README.md

```diff
@@ -48,37 +48,36 @@ blade itself.
 
 ## Create an Alert Logic Access Key
 
-Login and get an authentication token from the Alert Logic Cloud Insight product [AIMS API](https://console.product.dev.alertlogic.com/api/aims/).  From the command line use [curl](https://en.wikipedia.org/wiki/CURL) as follows (where `<user>` is your CloudInsight user and `<pwd>` is your CloudInsight password):
-
-```
-curl -X POST -v -u '<user>:<pwd>' https://api.global-services.global.alertlogic.com/aims/v1/authenticate
-```
-
-Make a note of the following fields returned in the response:
-
-   * USER ID
-   * ACCOUNT ID
-   * TOKEN
-
-Use the authentication token returned in the response to create access keys for the Azure application deployed in the next section.  Issue the following curl command (where `<TOKEN>` is the auth token, `<ACCOUNT_ID>` is the account ID, and `<USER_ID>` is the user ID returned above):
-
+From the Bash command line in [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/quickstart) run the following commands:
```

Comment:
I would include ... (where `<user>` is your CloudInsight user and `<pwd>` is your CloudInsight password):